content/team/anibalsolon.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "Anibal Solon Heinsfeld"
+date: 2022-01-26T17:32:40Z
+draft: false
+position : "Graduate Research Assistant"
+description : "Anibal is a graduate research assistant in the brainlife.io project. His focus is Machine Learning methods applied in Neuroscience, developing methods that account for reproducibility and reliability. Besides his academic career, Anibal contributed to several open-source projects and loves computer infrastructure and programming. He joined the University of Texas at Austin in 2019 and joined the brainlife.io team in 2021."
```

Comment:
How about the following?
"Anibal is a graduate student in the Pestilli Lab. His focus is Machine Learning methods applied in Neuroscience, developing methods that account for reproducibility and reliability. Anibal has contributed to several open-source projects (ADD A FEW HERE) and loves computer infrastructure and programming. He joined the University of Texas at Austin in 2019 and joined the brainlife.io team in 2021."

** See ADD A FEW HERE

Reply:
Thanks! I've added some.